pymc_marketing/clv/distributions.py

```diff
@@ -293,3 +294,190 @@ def logp(value, lam, p, T, T0):
             pt.all(T0 < T),
             msg="lam > 0, 0 <= p <= 1, T0 < T",
         )
+
+
+class ParetoNBDRV(RandomVariable):
+    name = "pareto_nbd"
+    ndim_supp = 1
+    ndims_params = [0, 0, 0, 0, 0]
+    dtype = "floatX"
+    _print_name = ("ParetoNBD", "\\operatorname{ParetoNBD}")
+
+    def make_node(self, rng, size, dtype, r, alpha, s, beta, T):
+
+        r = pt.as_tensor_variable(r)
+        alpha = pt.as_tensor_variable(alpha)
+        s = pt.as_tensor_variable(s)
+        beta = pt.as_tensor_variable(beta)
+        T = pt.as_tensor_variable(T)
+
+        return super().make_node(rng, size, dtype, r, alpha, s, beta, T)
+
+    def __call__(self, r, alpha, s, beta, T, size=None, **kwargs):
+        return super().__call__(r, alpha, s, beta, T, size=size, **kwargs)
+
+    @classmethod
+    def rng_fn(cls, rng, r, alpha, s, beta, T, size) -> np.array:
```

Comment:
I think the type hints should be ndarray. array is a function, ndarray is the class

```suggestion
    def rng_fn(cls, rng, r, alpha, s, beta, T, size) -> np.ndrray:
```